ivc/src/ivc/columns.rs

```diff
@@ -284,78 +293,90 @@ impl ColumnIndexer for IVCColumn {
     // This should be
     //   const N_COL: usize = std::cmp::max(IVCPoseidonColumn::N_COL, FECColumn::N_COL);
     // which is runtime-only expression..?
-    const N_COL: usize = IVCPoseidonColumn::N_COL;
+    // 333 is not enough
+    const N_COL: usize = 400;
```

Comment:
Where does this number come from?

Reply:
It's a stub. I don't have the exact number right now, but as the comment above says the number of columns for IVC should be maximum of Poseidon and FEC Addition.

Reply:
Okay, let's keep an eye on this in any case